packages/build-utils/test/fixtures/28-pnpm-7/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "22-pnpm",
```

Comment:
Lets remove everything except `scripts` and `dependencies`. And then also add `private: true`